_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+No wonder then, that the results the next day in the evening were brilliant:
+
+### Final presentations
+
+During the two days of the hackathon the teams mainly focused on pushing forward our new architecture Beyond, enhancing our current shop software ePages 6 and improving internal processes.
+The audience was quite impressed by the creative outcomes of every single team.
+
+For further developing our newest backend architecture some teams dedicated the two days on simplifying the interaction with the Beyond API with the help of the query language [GraphQl](http://graphql.org/learn/).
+Another team made it possible to enable merchants to interact with Beyond by only using natural spoken language by programming Amazon's voice service device [Alexa](https://developer.amazon.com/alexa).
+Yet another team created a dashboard app for visualizing KPIs for the merchant.
+
+One of the teams that was concerned about enhancing our current shop software developed a Amazon mass uploader to upload a bulk of products at once instead of merely one by one.
+
+Internal processes that were improved were, among others, the internal tool we use for our developer blog and REST API, as well as an automatization of the Linux setup for new colleagues starting at ePages.
+
+In the end everyone could vote for his or her favorite result.
+The first three winners were the lucky ones to get Amazon vouchers.
+
+The team that made the first place, was the one that developed a new stylish theme for our current core shop software ePages 6 as well as Beyond.
+New features for this theme were, for example, modern styling techniques, special fonts, a full screen slide bar, a featured products section, hovering effects and much more.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/ephack17?src=hash">#ephack17</a> WINNER TEAM of <a href="https://twitter.com/hashtag/epages?src=hash">#epages</a> <a href="https://twitter.com/hashtag/hackathon?src=hash">#hackathon</a> : Building a <a href="https://twitter.com/hashtag/theme?src=hash">#theme</a> for <a href="https://twitter.com/hashtag/epagesnow?src=hash">#epagesnow</a> and <a href="https://twitter.com/hashtag/beyond?src=hash">#beyond</a><br>Congrats! Well done! <a href="https://t.co/nHWxyWkOoV">pic.twitter.com/nHWxyWkOoV</a></p>&mdash; ePages Developers (@epagesdevs) <a href="https://twitter.com/epagesdevs/status/844973976208244736">March 23, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+But of course, in the end every single team contributed to the complete success of the ePages hackathon 2017 and this needed to be celebrated extensively {% emoji wink %}.
+
+### You want to join the hacking fun?
```

Comment:
Please leave that section out. We won't make it for today as the video has to be reviewed.
We will promote it separately.